config/base.yaml

```diff
@@ -13,6 +13,10 @@ remote_settings:
   default_bucket: main
   default_collection: quicksuggest
 
+elasticsearch:
+  connection:
+    Type: none
```

Comment:
nit: why's "Type" uppercase (we use a lowercase "type" in providers)?

Reply:
No reason, this is just a mistake.